File: frontend/app/src/main/java/com/example/geoquiz_frontend/data/remote/dtos/koth/LobbyInitialStateData.java
package com.example.geoquiz_frontend.data.remote.dtos.koth;

import java.util.List;
import java.util.UUID;

public class LobbyInitialStateData {
    private String lobbyId;
    private List<PlayerLobby> players;
    private int totalPlayers;

    public String getLobbyId() { return lobbyId; }
    public List<PlayerLobby> getPlayers() { return players; }
    public int getTotalPlayers() { return totalPlayers; }
}

File: backend/GeoQuiz_backend/GeoQuiz_backend/Application/DTOs/KingOfTheHill/LobbyInitialStateData.cs
using GeoQuiz_backend.Domain.Entities;

namespace GeoQuiz_backend.Application.DTOs.KingOfTheHill
{
    public class LobbyInitialStateData
    {
        public Guid LobbyId { get; set; }
        public List<PlayerLobby> Players { get; set; } = new();
        public int TotalPlayers { get; set; }
    }
}
